### PR TITLE
[RA2 Ch02]: Adding missing network requirements form the table of Ch 4.5

### LINKED ISSUE #1339: [RA2 Ch02]: Add requirements from the table of Chapter 4.5
[RM2 meeting on 2020.03.19](https://wiki.lfnetworking.org/display/LN/2020-03-19+-+%5BCNTT+-+RA2%5D+-+Meeting+Agenda+and+Minutes) agreed to add the missing requirements from the table of [Chapter 4.5](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/kubernetes/chapters/chapter04.md#45-cni-plugins) as requirements in a new, clarified form.

### PR DESCRIPTION
**Pull request by CsatariGergely**
Partly fixes #1339

### CODE REVIEW

**Comment by rabi-abdel** [comment]
@tomkivlin once this is approved by you, it can be merged.

File: doc/ref_arch/kubernetes/chapters/chapter02.md [hunk 1]
```diff
@@ -215,6 +215,8 @@ Reference to "Architecture" in this chapter refers to the NFVI Hardware (e.g. ph
 | `req.inf.ntw.14` | Network | The Architecture **must** support capabilities for integrating SDN controllers. |
 | `req.inf.ntw.15` | Network | The Architecture **should** support Service Mesh Interface (SMI). |
 | `req.inf.ntw.16` | Network | The Architecture **should**  support coexistence of multiple connection points in a pod. |
+| `req.inf.ntw.17` | Network | The Architecture **must** support cluster wide coordination of IP address assignement. |
```
Comment (tomkivlin) [inline doc/ref_arch/kubernetes/chapters/chapter02.md]: assignement -> assignment

File: doc/ref_arch/kubernetes/chapters/chapter02.md [hunk 2]
```diff
@@ -215,6 +215,8 @@ Reference to "Architecture" in this chapter refers to the NFVI Hardware (e.g. ph
...
+| `req.inf.ntw.18` | Network | The Architecture **should** support the usage of any interfaces with a Kubernetes Service if the interface is designed to work with Kubernetes Services. |
```
Comment (tomkivlin) [inline doc/ref_arch/kubernetes/chapters/chapter02.md]: usage -> use
interfaces -> interface

File: doc/ref_arch/kubernetes/chapters/chapter02.md (hunk 1 above)
Comment (CsatariGergely) [reply doc/ref_arch/kubernetes/chapters/chapter02.md]: Fixed

File: doc/ref_arch/kubernetes/chapters/chapter02.md (hunk 2 above)
Comment (CsatariGergely) [reply doc/ref_arch/kubernetes/chapters/chapter02.md]: Fixed

**Pull request merged**